packages/xrpc-server/src/types.ts

```diff
@@ -242,7 +241,7 @@ export class XRPCError extends Error {
   }
 
   get typeName(): string | undefined {
-    return ResponseTypeNames[this.type]
+    return ResponseType[this.type]
```

Comment:
No `changeset` required here since it will get a version bump from its dependency